.evergreen/run-mongodb-oidc-test.sh

```diff
@@ -49,7 +44,7 @@ if [ $OIDC_PROVIDER_NAME == "aws" ]; then
     export OIDC_ADMIN_PWD=$OIDC_ATLAS_PASSWORD
 
 elif [ $OIDC_PROVIDER_NAME == "azure" ]; then
-    if [ -z "${AZUREOIDC_AUDIENCE}" ]; then
+    if [ -z "${AZUREOIDC_AUDIENCE:-}" ]; then
```

Comment:
I'm not a sh expert. Is there a reason that you mix -n and -z? Isn't ! -n equivalent to -z?

Reply:
Yes, `! -n` is equivalent to `-z`, but I prefer brevity. 😄